src/components/Table/style.js

```diff
@@ -52,13 +48,18 @@ export const pageBut = css`
   &:hover {
     color: var(--global-white);
     background-color: var(--primary-80);
+    cursor: pointer;
   }
 `
 export const activeBut = css`
   color: var(--global-white);
   background-color: var(--primary-90);
 `
 
+export const noActiveBut = css`
```

Comment:
What happens if we hover this button? Should we disable also the hover effects? For example the cursor should not become pointer in this case